<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_DataPlayer</class>
 <widget class="QMainWindow" name="ihm_DataPlayer">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Generateur">
       <attribute name="title">
        <string>Génération de signal</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" rowspan="2">
         <widget class="QListWidget" name="liste_variables">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
          <property name="uniformItemSizes">
           <bool>false</bool>
          </property>
          <property name="selectionRectVisible">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="PB_refresh_liste">
          <property name="toolTip">
           <string>Mise à jour de la liste des variables disponibles</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>171</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QGroupBox" name="groupBox_ConfigGenerator">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Paramètrage</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="3">
            <widget class="QLineEdit" name="signal_name">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Chox du signal</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QToolButton" name="PB_choixSignal">
             <property name="toolTip">
              <string>Choix du fichier signal</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="dureeEchantillon_label">
             <property name="text">
              <string>Durée échantillon</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QSpinBox" name="dureeEchantillon">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> [msec]</string>
             </property>
             <property name="maximum">
              <number>9000000</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="nombreCycles_label">
             <property name="text">
              <string>Nombre de cycles</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="2">
            <widget class="QSpinBox" name="nombreCycles">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Valeur spéciale &quot;-1&quot; pour que le signal soit joué en boucle</string>
             </property>
             <property name="specialValueText">
              <string>En boucle</string>
             </property>
             <property name="suffix">
              <string/>
             </property>
             <property name="minimum">
              <number>-1</number>
             </property>
             <property name="maximum">
              <number>9000000</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2" colspan="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="PB_StartGeneration">
             <property name="toolTip">
              <string>Commence la génération du signal</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/player_play.png</normaloff>:/icons/player_play.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="PB_StopGeneration">
             <property name="toolTip">
              <string>Arrête la génération du signal</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/player_stop.png</normaloff>:/icons/player_stop.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Rejoueur">
       <attribute name="title">
        <string>Rejoueur de trace</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="player_label">
          <property name="text">
           <string>Player</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="playerNameListe">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Choix du player (possibilité de jouer plusieurs traces en même temps)</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="PB_AjoutPlayer">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Ajouter un nouveau player</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>95</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" rowspan="2" colspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Paramètrage</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0" colspan="5">
            <widget class="QLineEdit" name="trace_name">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Chox du fichier de trace</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QToolButton" name="PB_choixTrace">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Choix du fichier signal</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="dureeStepTrace_label">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Durée du step</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="4">
            <widget class="QSpinBox" name="dureeStepTrace">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Vitesse de l'animation. Valeur spéciale &quot;-1&quot; pour indiquer que la durée du step est prise du fichier de trace</string>
             </property>
             <property name="specialValueText">
              <string>Fichier trace</string>
             </property>
             <property name="suffix">
              <string> [msec]</string>
             </property>
             <property name="minimum">
              <number>-1</number>
             </property>
             <property name="maximum">
              <number>9000000</number>
             </property>
             <property name="value">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="nombreCycles_label_2">
             <property name="text">
              <string>Nombre de cycles</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="4">
            <widget class="QSpinBox" name="nombreCyclesTrace">
             <property name="toolTip">
              <string>Valeur spéciale &quot;-1&quot; pour indiquer que le signal est joué en boucle</string>
             </property>
             <property name="specialValueText">
              <string>En boucle</string>
             </property>
             <property name="suffix">
              <string/>
             </property>
             <property name="minimum">
              <number>-1</number>
             </property>
             <property name="maximum">
              <number>9000000</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="PB_StepBackwardTrace">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Joue le pas précédent</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/backward.png</normaloff>:/icons/backward.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QPushButton" name="PB_StartTrace">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Commence la trace</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/player_play.png</normaloff>:/icons/player_play.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="PB_StopTrace">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Arrêt de jouer la trace</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/player_stop.png</normaloff>:/icons/player_stop.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="4" colspan="2">
            <widget class="QPushButton" name="PB_StepForwardTrace">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Joue le pas suivant</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/forward.png</normaloff>:/icons/forward.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="6">
            <widget class="QSlider" name="sliderStepNumTrace">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="PB_StartAllTraces_Synchro">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Commence la trace de manière sychronisée sur tous les players</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/icons/player_play.png</normaloff>:/icons/player_play.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="PB_StopAllTraces_Synchro">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Arrêt de jouer la trace de manière sychronisée sur tous les players</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/icons/player_stop.png</normaloff>:/icons/player_stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>95</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>377</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
